### PR TITLE
More Reformatting 

### PR DESCRIPTION
**Pull request by ErinCain**

### CODE REVIEW

File: data-raw/watershed/cow/cow_creek.rmd [hunk 1]
```diff
@@ -16,16 +16,13 @@ library(cvpiaHabitat)
 Instream habitat suitability relationships provided for the Voluntary Agreement process.
```
Comment (thesadie) [inline data-raw/watershed/cow/cow_creek.rmd]: we don't have new data coming for cow creek, so change to none planed

File: data-raw/watershed/feather/feather_river.rmd [hunk 2]
```diff
@@ -63,44 +60,30 @@ feather_river_instream <- feather_raw %>%
          ST_spawn_wua,
          watershed)
 
-knitr::kable(align ="c", head(feather_river_instream, 5))
+knitr::kable(align ="c", head(feather_river_instream, 5), caption = "Header Descriptions: flow_cfs = flow in cubic feet per second, FR_spawn_wua = Fall Run Spawning wua, ST_fry_wua = Steelhead Fry wua, ST_juv_wua = Steelhead Juvenile wua, ST_spawn_wua = Steelhead Spawning wua")
```
Comment (thesadie) [inline data-raw/watershed/feather/feather_river.rmd]: I see more things in the header description than columns in the table (ST rearing), plz double check

File: data-raw/watershed/butte/butte_creek.rmd [hunk 3]
```diff
@@ -17,16 +17,10 @@ None Planned
...
 
-[FERC relicensing of DeSabla](https://s3-us-west-2.amazonaws.com/cvpiahabitat-r-package/cvpia-sit-model-inputs/DeSabla2008ButteIFIM.pdf) (APPENDIX E6.3.2.6-J1)
+Instream spawning and rearing habitat for Fall Run Chinook Salmon in Butte Creek are based on data from the FWS instream 2003 study and the FERC relicensing of DeSabla 2008 study. The spawning habitat data is from the FWS study and was collected using a RIVER2D 2-dimensional hydraulic and habitat model and can be found on pages 77-78 of the study. The rearing data came from DeSabla 2008 in the lower butte section of the appendix (APPENDIX E6.3.2.6-J1). Instream rearing habitat data was provided in a spreadsheet (see '~cvpiaHabitat/data-raw/mark_gard_data/IFIMWUA.xlsx'). Mark Gard instructed us to use cells A71:C101 for Fall Run Chinook rearing data. The spawning data was compiled in a spreadsheet (see '~cvpiaHabitat/data-raw/watershed/butte/data/butte_creek_data.csv). The spawning data was then scaled by reach length. No Fall Fun Chinook Salmon Spawning data was collected so Spring Run Chinook Salmon Spawning is used as a proxy for Fall Run Chinook Salmon Spawning. Units are in square feet per 1000 feet.
```
[the remark this answers is not in the document]
Comment (ErinCain) [reply data-raw/watershed/butte/butte_creek.rmd]: done

File: data-raw/watershed/butte/butte_creek.rmd [hunk 4]
```diff
@@ -60,46 +54,56 @@ butte_creek_spawning <- tibble(
...
 #usethis::use_data(butte_creek_instream, overwrite = TRUE)
 ```
 
-*...with 46 more rows.*
+### Spawning Data
+```{r, echo = FALSE}
+knitr::kable(align = 'c', head(butte_creek_spawning %>% 
+                                 mutate(watershed = 'Butte Creek'), 5), caption = 
+                                "Header Descriptions: flow_cfs = flow in cubic feet per second, 
+                                FR_spawn_wua = Fall Run Chinook Spawning wua")
```
[the remark this answers is not in the document]
Comment (ErinCain) [reply data-raw/watershed/butte/butte_creek.rmd]: fixed

File: data-raw/watershed/butte/butte_creek.rmd [hunk 5]
```diff
@@ -60,46 +54,56 @@ butte_creek_spawning <- tibble(
...
+```
 
-### Spawning WUA 
-The instream spawning habitat weighted usable areas for Fall Run Chinook Salmon in Butte Creek.
```
[the remark this answers is not in the document]
Comment (ErinCain) [reply data-raw/watershed/butte/butte_creek.rmd]: changed

File: data-raw/watershed/cow/cow_creek.rmd (hunk 1 above)
Comment (ErinCain) [reply data-raw/watershed/cow/cow_creek.rmd]: done

File: data-raw/watershed/feather/feather_river.rmd (hunk 2 above)
Comment (ErinCain) [reply data-raw/watershed/feather/feather_river.rmd]: fixed

**Pull request merged**